packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -399,8 +432,27 @@ class ScrollableBase extends Component {
 
 	onKeyDown = (ev) => {
 		this.animateOnFocus = true;
-		if (!Spotlight.getPointerMode() && (isPageUp(ev.keyCode) || isPageDown(ev.keyCode)) && !ev.repeat && this.hasFocus()) {
-			this.scrollByPage(ev.keyCode);
+
+		if (!Spotlight.getPointerMode() && !ev.repeat && this.hasFocus()) {
```

Comment:
I don't think that we need `!Spotlight.getPointerMode()` condition because it should scroll when pressing a page up or down key in pointer mode.

Reply:
So, was it a bug that calling `this.scrollByPage` only when 5-way key mode?

Reply:
I will check this when I update with the latest `develop` branch.

Reply:
Okay, it looks like a different issue from this PR. It will be handled with a new ticket.